implementations/CMIP6-UofT/add_CMIP6.py

```diff
@@ -39,11 +40,14 @@ def __init__(
             print(item)
         super().__init__(stac_host, data_loader, config_filename)
 
-    def process_stac_item(self):  # noqa N802
+    def handle_ingestion_error(self, error: str, item_name: str, item_data: MutableMapping[str, Any]):
+        pass
+
+    def create_stac_item(self, item_name: str, item_data: MutableMapping[str, Any]) -> MutableMapping[str, Any]:
```

Comment:
In practice, I suspect this method will both ingest and validate in one step, but let's keep that structure.